build/commands/lib/config.js

```diff
@@ -134,7 +134,7 @@ const Config = function () {
   this.rewardsGrantDevEndpoint = getNPMConfig(['rewards_grant_dev_endpoint']) || ''
   this.rewardsGrantStagingEndpoint = getNPMConfig(['rewards_grant_staging_endpoint']) || ''
   this.rewardsGrantProdEndpoint = getNPMConfig(['rewards_grant_prod_endpoint']) || ''
-  this.chromePgoPhase = 0
+  this.chromePgoPhase = getNPMConfig(['chrome_pgo_phase']) || 0
```

Comment:
I've taken a look at `dcheck_always_on` and have done it in a similar way: using npm config variable that can be set on CI.

Reply:
same as https://github.com/brave/brave-core/pull/11598/files#r770858667 we don't want to set a default here, we only want to use this if we are explicitly overriding the default in gn. The only time we should set defaults here is when we always want to override the default set by upstream

Reply:
I don't think this makes sense as an npm config, the only time we'd override the default is for testing which would make more sense on the command line

Reply:
Removed.
BTW sometimes it's hard to calculate if we need to override chromium default or not, because we need to duplicate the part of upstream logic here.